src/objects.js

```diff
@@ -1,41 +1,47 @@
 const createPerson = (name, age) => {
-  // your code here
+  return { name: name, age: age };
 };
 
 const getName = (object) => {
-  // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
-  // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+  return object.hasOwnProperty(property);
 };
 
 const isOver65 = (person) => {
-  // your code here
+  return person.age > 65;
 };
 
 const getAges = (people) => {
-  // your code here
+  return people.map((person) => {
+    return person.age;
+  });
 };
 
 const findByName = (name, people) => {
-  // your code here
+  return people.filter(person => name === person.name)[0];
 };
 
 const findHondas = (cars) => {
-  // your code here
+  return cars.filter(car => car.manufacturer === 'Honda');
 };
 
 const averageAge = (people) => {
```

Comment:
Your test is returning the value `NaN` instead of what is expected. It looks like the accumulator hasn't been defined correctly on the `reduce` function. This could also have been solved with a `for` loop or other array methods. 
Have a look at - https://www.freecodecamp.org/news/a-guide-to-the-reduce-method-in-javascript-f47a7da511a9/

We also prefer to have some more explicit names for our variables - `acc` or `accumulator` is better than `a`, `person` better than `c`.